### PR TITLE
[#3910] feat (trino-connector): Support load plugin in Trino DevelopmentServer startup mode

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

(Please outline the changes and how this PR fixes the issue.)

Support load plugin in Trino DevelopmentServer startup mode

Fix: #3910

### Does this PR introduce _any_ user-facing change?

Update docs

### How was this patch tested?

Manually test

### CODE REVIEW

File: trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java [hunk 1]
```diff
@@ -140,4 +159,50 @@ static class ConfigEntry {
...
+
+    public TrinoConfig() {
+      this.properties = new Properties();
+      try {
+        String configFileName = System.getProperty("config");
+        properties.load(new FileInputStream(configFileName));
+        if (properties.containsKey(TRINO_CATALOG_STORE)
+            && !properties
+                .getProperty(TRINO_CATALOG_STORE)
+                .equals(TRINO_CATALOG_STORE_DEFAULT_VALUE)) {
```
Comment (yuqi1129) [inline trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java:179]: So what the value of `TRINO_CATALOG_STORE` is `File`? the final result is true or false here?

Comment (diqiu50) [reply trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java:188]: Maybe `file` or `memory`. if it's file return true

Comment (yuqi1129) [reply trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java:188]: I mean whether it is case sensitive such as `file` or `file`.

Comment (diqiu50) [reply trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java:188]: It's false, it's case insensitive

**Review by yuqi1129 — APPROVED** [review]

**Pull request merged**